### PR TITLE
feat: case width

### PR DESCRIPTION
**Pull request by lundeeell**
Aligning content with /case widths for title/ingress

### CODE REVIEW

**Review by izabellalarsson — APPROVED** [review]
Think it looks good! 👍

**Review by izabellalarsson — COMMENTED** [review]
Just saw that "Teknik" imgs dosen't follow the wrapper.

**Comment by lundeeell** [comment]
> Just saw that "Teknik" imgs dosen't follow the wrapper.

oh, will fix!

File: sapper/static/global.css [hunk 1]
```diff
@@ -63,34 +63,44 @@ body {
   font-size: 1rem;
 }
 
+.post div,
```
Comment (izabellalarsson) [inline sapper/static/global.css:66]: We have some styling on this div in tailwind so maybe extract that styling or move it into here. I would vote for doing this styling in tailwind :)

Comment (izabellalarsson) [reply sapper/static/global.css:66]: Just so we just handle the html blob in plane css

Comment (lundeeell) [reply sapper/static/global.css:66]: Oh I see! Agreed!

**Comment by izabellalarsson** [comment]
👍

**Pull request merged**